### DISCUSSION TITLE
v0.13.10

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1640] [Releases]
https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#01310---2024-11-30

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.13.10'>v0.13.10</a>.</em>

**Comment by m-ueberall** [comment]
Not sure where to put it, so here goes:
If possible, could we skip v0.64.0 of `libsignal-client` in a future release? While v0.63.0 and v0.64.1 build without any problems for a large number of architectures, more than 20 builds failed locally for v0.64.0 (`ubuntu1310_armv7,ubuntu1604_mips,ubuntu1604_mips64,ubuntu1604_powerpc,ubuntu1604_ppc64,ubuntu1604_ppc64el,ubuntu1604_s390x,ubuntu1604_sparc64,ubuntu1804_mips,ubuntu1804_mips64,ubuntu1804_powerpc,ubuntu1804_ppc64,ubuntu1804_ppc64el,ubuntu1804_s390x,ubuntu1804_sparc64,ubuntu2004_loong64,ubuntu2004_mips,ubuntu2004_mips64,ubuntu2004_powerpc,ubuntu2004_ppc64,ubuntu2004_ppc64el,ubuntu2004_s390x,ubuntu2004_sparc64`).

**Reply by m-ueberall** [reply]
**Update:** The above comment can be _ignored_; due to a brown-paper-bag bug, the build script downloaded the wrong git branch of `libboring` to be patched… :sweat:

**Comment by nomansial** [comment]
After sending 4 to 5 messages using signal-cli v0.13.10 Im getting error **"failed to send message due to rate limiting"**
@AsamK  Could you please assist how to fix this issue?
